second-state/Meta-Llama-3-70B-Instruct-GGUF.json

```diff
@@ -0,0 +1,220 @@
+{
+    "model_id": 8341,
+    "id": "second-state/Meta-Llama-3-70B-Instruct-GGUF",
+    "status": "published",
+    "trace_status": "tracing",
```

Comment:
No need for this. We just need index files for published / verified model cards.